dotCMS/src/main/java/com/dotcms/analytics/content/ReportResponse.java

```diff
@@ -0,0 +1,24 @@
+package com.dotcms.analytics.content;
+
+import com.dotcms.analytics.model.ResultSetItem;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * Represents the response of a report.
+ * @author Jose Castro
+ * @since Sep 13th, 2024
+ */
+public class ReportResponse implements Serializable {
+
+    private final List<ResultSetItem> results;
+
+    public ReportResponse(final List<ResultSetItem> results){
```

Comment:
why don't you used the "CubeJSResultSet" instead of this class?

Reply:
The idea behind that is to avoid using class/variables names that include the word `CubeJS` in them and replace or wrap them into more generic classes.